device/globals/target.gd

```diff
@@ -0,0 +1,10 @@
+extends Position2D
```

Comment:
Oh, right. It doesn't extend any of the other scripts. Probably makes less sense to stub `set_state` then.